omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/MockServerLogActionWrapper.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web;
+
+import org.openmrs.module.webservices.helper.ServerLogActionWrapper;
+import java.util.ArrayList;
+import java.util.List;
+
+/***
```

Comment:
Did you format your code?

Reply:
Sorry for the mistake, I will make it better